spaceship.zsh

```diff
@@ -84,6 +84,8 @@ SPACESHIP_DIR_SUFFIX="${SPACESHIP_DIR_SUFFIX:="$SPACESHIP_PROMPT_DEFAULT_SUFFIX"
 SPACESHIP_DIR_TRUNC="${SPACESHIP_DIR_TRUNC:=3}"
 SPACESHIP_DIR_COLOR="${SPACESHIP_DIR_COLOR:="cyan"}"
 
+SPACESHIP_DIR_TRUNC_IN_GIT=true;
```

Comment:
Yes, this was just to show the toggle/option. If that would be something we want?